packages/pages/src/App.js

```diff
@@ -92,15 +86,52 @@ export default provideState({
         await this.effects.refreshEntries();
       }
     },
-    async changeName(
+    async submit(effects, event) {
+      event.preventDefault();
+
+      var bodyToSend = "";
+      if (this.state.id === "") {
+        bodyToSend = format.request(0, "createEntry", {
+          name: this.state.name,
+          content: this.state.content,
+        });
+      } else {
+        bodyToSend = format.request(0, "updateEntry", {
+          id: this.state.id,
+          name: this.state.name,
+          content: this.state.content,
+        });
+      }
+      const response = await fetch("/api/", {
+        method: "post",
+        body: bodyToSend,
+      });
+      const parsed = parse(await response.text());
```

Comment:
You should use the new `parse.result` (you may need to update `json-rpc-protocol|).